### PR TITLE
Rename Windows Metrics(swap -> virtual memory)

### LINKED ISSUE #12105: [Bug] Windows Metrics Name -- "Swap" => "Virtual Memory"
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

Windows metrics that include `swap` are from 'virtual memory', but swap is not equal to virtual memory.

With [MSINFO32.exe](https://www.techtarget.com/searchenterprisedesktop/tip/Learn-native-Windows-10-virtual-memory-management-methods), we could see resources' current status, including physical memory and virtual memory and page file size.

As I know, [Virtual Memory] = [Physical Memroy] + [Swap].
And Zabbix Agent read swap number is page file.

### What you expected to happen

Rename metrics:
- memory_swap_free => memory_virtual_memory_free 
- memory_swap_total => memory_virtual_memory_total
- memory_swap_percentage => memory_virtual_memory_percentage

(There is two 'memory' in each metry, so may have another better name.)

### How to reproduce

remove document or table - ui_template_Windows-Service - in database, or using empty database.

### Anything else

_No response_

### Are you willing to submit a pull request to fix on your own?

- [X] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by lagagain**
### Fix #12105 
- [ ] Add a unit test to verify that the fix works.
- [x] Explain briefly why the bug exists and how to fix it.
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #12105.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).
- [ ] ~~Update UI template for Windows Service~~

### CODE REVIEW

File: oap-server/server-starter/src/main/resources/ui-initialized-templates/os_windows/windows-service.json [hunk 1]
```diff
@@ -125,6 +121,13 @@
           {
             "label": "available"
           }
+        ],
+        "metrics": [],
+        "metricMode": "Expression",
+        "typesOfMQE": [
+            "TIME_SERIES_VALUES",
+            "TIME_SERIES_VALUES",
+            "TIME_SERIES_VALUES"
```
Comment (wu-sheng) [inline oap-server/server-starter/src/main/resources/ui-initialized-templates/os_windows/windows-service.json:130]: @lagagain Please use the latest UI to export, it is changed recently.

**Comment by wu-sheng** [comment]
With your metric name changes, the e2e tests may need to change, or CI will fail.

**Comment by wu-sheng** [comment]
https://github.com/apache/skywalking/tree/master/test/e2e-v2/cases/win

Here is the e2e, could you verify the e2e as well.

And follow the PR template to update changes.md.

File: oap-server/server-starter/src/main/resources/ui-initialized-templates/os_windows/windows-service.json (hunk 1 above)
Comment (lagagain) [reply oap-server/server-starter/src/main/resources/ui-initialized-templates/os_windows/windows-service.json:130]: Are there documents about ui-template settings?  Where I could found?

----------

I changed UI init setting with Download from Database after I manually changed the UI with version 9.7.
I can't understand what the setting means.

Comment (wu-sheng) [reply oap-server/server-starter/src/main/resources/ui-initialized-templates/os_windows/windows-service.json:130]: This is UI settings, once you changed that, and Database is empty, they would be reimported.

And you cam change it from UI in local UI with edit mode, and export it to generate the JSON file.

It is changed in v10 dev about this JSON file, so, use the latest of the codes(not 9.7) and edit the changes.

Comment (wu-sheng) [reply oap-server/server-starter/src/main/resources/ui-initialized-templates/os_windows/windows-service.json:130]: You should either use the latest UI or at least not change this part, and keep rename only.

Comment (lagagain) [reply oap-server/server-starter/src/main/resources/ui-initialized-templates/os_windows/windows-service.json]: I remove the commit.

File: oap-server/server-starter/src/main/resources/ui-initialized-templates/os_linux/linux-service.json [hunk 2]
```diff
@@ -339,7 +339,7 @@
             "showUnit": true
           },
           "expressions": [
-            "avg(meter_vm_memory_swap_percentage)/100"
```
Comment (wu-sheng) [inline oap-server/server-starter/src/main/resources/ui-initialized-templates/os_linux/linux-service.json:342]: Isn't the precision of the value correct?

Comment (wu-sheng) [reply oap-server/server-starter/src/main/resources/ui-initialized-templates/os_linux/linux-service.json:342]: Usually precision is using 4 digital places precision. Is the original data using 2 deci

Comment (lagagain) [reply oap-server/server-starter/src/main/resources/ui-initialized-templates/os_linux/linux-service.json:342]: This Card uses "%" unit. (line: 350)
And the value is already * 100 at the [formula/rule](https://github.com/apache/skywalking/blob/c551a9ddb68bc34f44f00fe0dc5b5cb4959e0d19/oap-server/server-starter/src/main/resources/otel-rules/vm.yaml#L62).

    exp: 100 - ((node_memory_SwapFree_bytes * 100) / node_memory_SwapTotal_bytes)

See an example result:
圖片

`node_memory_SwapTotal_bytes` is `3930.996`.
`node_memory_SwapFree_bytes` is `3848.723`

    100 - (3848.723 * 100) / 3930.996 = 2.092930138

But the card display `0.02%`.

Expect is `2%`: 
圖片

Compare [CPU Usage](https://github.com/apache/skywalking/blob/c551a9ddb68bc34f44f00fe0dc5b5cb4959e0d19/oap-server/server-starter/src/main/resources/ui-initialized-templates/os_linux/linux-service.json#L400).

It also [* 100 at rule](https://github.com/apache/skywalking/blob/c551a9ddb68bc34f44f00fe0dc5b5cb4959e0d19/oap-server/server-starter/src/main/resources/otel-rules/vm.yaml#L38), and display "%" unit.

圖片

**Review by wu-sheng — APPROVED** [review]

**Review by wankai123 — APPROVED** [review]

**Pull request merged**